script.js

```diff
@@ -0,0 +1,122 @@
+let rootDiv = document.getElementById("root");
+let gridsize = 16; //default
+let clr = document.getElementById("clr");
+let radioB1RandColors = document.getElementById("radioBut1RandomColors");
+let radioB2RandColor = document.getElementById("radioBut2RandomColor");
+let radioB3PickColor = document.getElementById("radioBut3PickColor");
```

Comment:
there's something interesting about the appearance of mode option, putting them that way horizontally may confuse the user. it would be better if you put the three options of the mode under each other vertically. that way the user can realize what are the available modes.
![image](https://user-images.githubusercontent.com/56412800/98304413-9a71c680-1fc8-11eb-96b3-d708e31d7fb2.png)

Reply:
but no space vertically